Library/Homebrew/cask/audit.rb

```diff
@@ -301,11 +301,16 @@ def check_download
     def check_appcast_contains_version
       return unless check_appcast?
       return if cask.appcast.to_s.empty?
+      return if cask.appcast.configuration == :no_check
 
       appcast_stanza = cask.appcast.to_s
-      appcast_contents, = curl_output("--max-time", "5", appcast_stanza)
+      appcast_contents, = curl_output("-L", "--max-time", "5", appcast_stanza)
```

Comment:
Can you use `--location` here as it's slightly more obvious what it does.

Reply:
thanks! done